lsp/nls/src/linearization/mod.rs

```diff
@@ -24,7 +24,8 @@ pub mod building;
 pub mod completed;
 pub mod interface;
 
-pub type Environment = nickel_lang::environment::Environment<Ident, usize>;
+pub type ItemID = (FileId, usize);
```

Comment:
Nitpick: we could use a struct here, like `struct ItemId { file_id: FileId, index: usize }` (or something like that).

Also, `ItemID` should be `ItemId` in CamelCase, I think.